Homework/Van_Le/HappyNumber/Solution.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace HappyNumber
+{
+    public static class HappyNumber
+    {
+        public static bool IsHappy(int n)
+        {
+            List<int> usedNumbers = new List<int>();
```

Comment:
can improve to O(1) space using FLoyd algorithm?

Reply:
I have updated it, can you approve this pull request?